pkgs/development/python-modules/inkex/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  buildPythonPackage,
+  pkgs,
```

Comment:
could we import just inkscape here rather than all of pkgs?

Reply:
Yup, this was remnant from when I defined it in `python-modules.nix`.